### ISSUE TITLE
Wrong realm for Webdav auth in 9.0

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#26720]
### Steps

1. `curl -D - -X PROPFIND http://localhost/owncloud/remote.php/webdav/`
1. Check headers

9.0:
```
WWW-Authenticate: Basic realm="sabre/dav"
```

All other OC versions:
```
WWW-Authenticate: Basic realm="ownCloud"
```

Seems to be a regression, might be bisectable.

Discovered when backporting header test: https://github.com/owncloud/core/pull/26712

@DeepDiver1975 @SergioBertolinSG how critical ?

**Comment by PVince81** [comment]
I'll quickly bisect this

**Comment by PVince81** [comment]
v9.0.0 affected too, possibly due to sabre upgrade

**Comment by PVince81** [comment]
Introduced by 4bb346a1e6fe2407dd96d35a1d6e30af29ff6432 from https://github.com/owncloud/core/pull/20692

Now why does it work properly in stable9.1 ? Maybe it was fixed later on.
Let's do a reverse-bisect to find out...

**Comment by DeepDiver1975** [comment]
I'm pretty sure I did fix this - looks like a backport is missing ...

**Comment by PVince81** [comment]
found it "fix the real", will backport: cf06b17df15d6de83d2893cbe08880bbd9dd5c2c

**Comment by PVince81** [comment]
Fix/backport is here: https://github.com/owncloud/core/pull/26721
